database/migrations/2022_03_18_195003_create_news_table.php

```diff
@@ -11,7 +11,7 @@ public function up(): void
     {
         Schema::create("news", function (Blueprint $table): void {
             $table->id();
-            $table->foreignId("author_id")->constrained()->onDelete("cascade");
+            $table->foreignId("user_id")->constrained()->onDelete("cascade");
```

Comment:
Migration has rename method. Please do actions like that in new one migration file.

Reply:
It will not work with SQLite. :(